main/test/flix/Test.Exp.ForYield.flix

```diff
@@ -3,42 +3,42 @@ namespace Test/Exp/ForYield {
     @test
     def testForYield01(): Bool =
         let l =
-            for (x <- 1 :: 2 :: 3 :: Nil)
+            forM (x <- 1 :: 2 :: 3 :: Nil)
```

Comment:
Do we also rename the file?

Reply:
Done in separate PR, so merging main will do the trick

Reply:
But it still called ForYield.flix or you mean a separate PR is coming?

Reply:
I'll push the changes, but on main / master the file has been renamed

Reply:
Ah okay